lib/killbill_client/models/account.rb

```diff
@@ -297,6 +297,65 @@ def all_custom_fields(object_type, audit = 'NONE', options = {})
                        options,
                        CustomField
       end
+      def blocking_states(blocking_state_types, blocking_state_svcs, audit = 'NONE', options = {})
+        params = {}
+        params[:blockingStateTypes] = blocking_state_types if blocking_state_types
+        params[:blockingStateSvcs] = blocking_state_svcs if blocking_state_svcs
+        params[:audit] = audit
+        self.class.get "#{KILLBILL_API_ACCOUNTS_PREFIX}/#{account_id}/block",
+                       params,
+                       options
+      end
+
+      def block_account(state_name, service, block_change, block_entitlement, block_billing, requested_date = nil, user = nil, reason = nil, comment = nil, options = {})
```

Comment:
Maybe rename it to `set_blocking_state`  for consistency (see lib/killbill_client/models/bundle.rb).